apps/useradmin/src/useradmin/templates/layout.mako

```diff
@@ -104,6 +104,22 @@ def is_selected(section, matcher):
 <%def name="commons()">
   <link href="${ static('useradmin/css/useradmin.css') }" rel="stylesheet">
   <script>
+    var entityMap = {
+     '&': '&amp;',
+     '<': '&lt;',
+     '': '&gt;',
+     '"': '&quot;',
+     "'": '&#39;',
+     '/': '&#x2F;',
+     '`': '&#x60;',
+     '=': '&#x3D;'
+    };
+
+    function escapeHtml(string) {
+       return String(string).replace(/[&<"'`=\/]/g, function(s) {
+        return entityMap[s];
+       });
+    }
```

Comment:
Use hueUtils.deXSS(str) instead

Reply:
Code changed as per review comment

Reply:
i have changed it to : hueUtils.escapeOutput